locales/cs.json

```diff
@@ -1,60 +1,60 @@
 {
-    "alphabetically": "Abecedně",
```

Comment:
Indents?

Reply:
I reset the file to master, since there's no change in it (We should really get Weblate to respect indent size, though...)